includes/admin/class-sensei-learners-main.php

```diff
@@ -107,33 +180,32 @@ function get_columns() {
 	 * @since  1.7.0
 	 * @return array $columns, the array of columns to use with the table
 	 */
-	function get_sortable_columns() {
-		$columns = array();
+	public function get_sortable_columns() {
+
 		switch ( $this->view ) {
 			case 'learners':
 				$columns = array(
 					'title' => array( 'title', false ),
 				);
 				break;
-
 			case 'lessons':
-				$columns = array(
-					'title'   => array( 'title', false ),
-					'updated' => array( 'post_modified', false ),
-				);
-				break;
-
 			default:
 				$columns = array(
 					'title'   => array( 'title', false ),
 					'updated' => array( 'post_modified', false ),
 				);
 				break;
 		}
-		// Backwards compatible
-		if ( 'learners' == $this->view ) {
-			$columns = apply_filters( 'sensei_learners_learners_columns_sortable', $columns, $this );
+		// Backwards compatible.
+		if ( 'learners' === $this->view ) {
+			$columns = apply_filters_deprecated(
+				'sensei_learners_learners_columns_sortable',
+				[ $columns, $this ],
+				'3.0.0',
+				'sensei_learners_default_columns'
```

Comment:
Should the replacement hook be `sensei_learners_default_columns_sortable` here?

Reply:
https://github.com/Automattic/sensei/pull/2988